buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -31,18 +25,18 @@
   {%- elif type == "double" -%}
     doubleKey
   {%- else -%}
-    {{lowerFirst(type)}}Key
+    {{type | to_camelcase(False)}}Key
   {%- endif -%}
 {%- endmacro %}
 {%- macro print_value(type, value) -%}
   {{ "\"" if type == "String"}}{{value}}{{ "\"" if type == "String"}}
 {%- endmacro %}
-{%- macro upFirst(text) -%}
-  {{ text[0]|upper}}{{text[1:] }}
-{%- endmacro %}
-{%- macro lowerFirst(text) -%}
-  {{ text[0]|lower}}{{text[1:] }}
-{%- endmacro %}
+{%- set filtered_attributes = attributes_and_templates | select(filter) | list %}
```

Comment:
you might want to exclude certain namespaces (e.g. `aspnetcore` is not particularly helpful on Java)

Here's a basic [example](https://github.com/open-telemetry/opentelemetry-python/blob/cbebd89c6da683c0516665a929774062ae93b42d/scripts/semconv/templates/semantic_attributes.j2#L1)

Reply:
Agreed. I opened a PR but think we should address this separately.